docs/src/prototypes/ComponentArchitecture/DemoTextControl.tsx

```diff
@@ -0,0 +1,18 @@
+import { compose } from '@fluentui/react-theming'
+import { BaseTextControl } from './BaseTextControl'
+import { defaultTheme } from './theme'
+
+export const DemoTextControl = compose(BaseTextControl, {
+  defaultTheme,
+  tokens: {
+    bg: (_: any, t: any) => t.colors.brand,
+  },
+  styles: t => {
```

Comment:
From what I can see there's some typing enforcement missing here that I assume will be added later, right?

Reply:
Exactly.